media/screenshot-fixtures/magic-assert-buffers.js

```diff
@@ -1,4 +1,5 @@
 import test from 'ava';
+import {Buffer} from 'safe-buffer';
```

Comment:
Let's not worry about this file, it's used by maintainers so we won't run it on Node.js 4 😉